src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -166,6 +166,12 @@ DlgPrefDeck::DlgPrefDeck(QWidget * parent, MixxxMainWindow * mixxx,
     connect(checkBoxSeekToCue, SIGNAL(toggled(bool)),
             this, SLOT(slotJumpToCueOnTrackLoadCheckbox(bool)));
 
+    // Automatically assign a color to new hot cues
+    m_bAssignHotcueColors = m_pConfig->getValue(ConfigKey("[Controls]", "AutoHotcueColors"), false);
```

Comment:
I thought about enabling this feature by default to let users notice. But on the other hand that might confuse and distract existing users. Let's be conservative and let users discover it for themselves.